### ISSUE TITLE
Question: How to not automatically lint files with (Kanban) in the Name

### ISSUE THREAD

**Issue by MMoMM-org** [platers/obsidian-linter#1244]
Atm I have the following regex in the Files to Ignore:
```
.*\(Kanban\)$
```
but Linter keeps on linting... (I have Lint on focused file change on)

Filenames are "filename (Kanban)".

Any Idea?

**Comment by pjkaufman** [comment]
Hey @MMoMM-org . My first guess is that the regex used is incorrect because it does not account for the file extension of the files you are trying to ignore. It looks like you are saying with the regex that the file path must not end in `(Kanban)`, but that would mean that there is no file extension attached. So I would either
- Change the regex to `.*\(Kanban\).*$`
- Change the regex to `.*\(Kanban\)\.md$` if you need the granularity and know that the file extension is `.md`

Does that help?

**Comment by MMoMM-org** [comment]
Yes... thanks... as linter only works on notes I wasn't expecting this reason.
